data/apps.ts

```diff
@@ -210,6 +211,12 @@ export const apps: appsList = {
       paid: true,
       hidden_from_all: true,
     },
+    {
+      released_on: "Aug 10, 2021",
+      name: "TootRain (Mac)",
```

Comment:
please remove (Mac) from the name - I know some other apps had that, but that was for apps which also had iOS versions, and I think we want a different way to show desktop platform in the future. Thanks.